### DISCUSSION TITLE
How to prevent user from accessing root folders via URL

### DISCUSSION THREAD

**Discussion by cngirwa** [LiveHelperChat/livehelperchat#1887] [Q&A] [answered]
Hello! I'm currently using v.4.06

When the user navigates to > https://livechat.mydomain.com/index.php/site_admin/cache/ 
the system redirects the user to > https://livechat.mydomain.com/index.php/site_admin/user/login/(r)/some_token 

However, when the user navigates to for example > https://livechat.mydomain.com/cache/
the system displays an error message 

> Forbidden
You don't have permission to access this resource.

Is it possible to redirect the user to the default URL when he/she tries any forbidden resource?

**Comment by remdex** [comment]
https://doc.livehelperchat.com/docs/development/remove-index-php

**Comment by cngirwa** [comment, marked as answer]
Found an answer to this question:

1. Make sure you are signed in and navigate to Settings
2. Change setting/settings.ini.php `'force_virtual_host' => true`
3. Clean the cache in the back office (about 2 or 3 times) until you see the error page
4. Edit .htaccess as suggested [here](https://doc.livehelperchat.com/docs/development/remove-index-php/)
5. Enable rewrite mod as suggested [here](https://stackoverflow.com/questions/869092/how-to-enable-mod-rewrite-for-apache-2-2)
6. Enable headers using` a2enmod headers `

Try to access forbidden pages, the system should redirect just fine
